guides/user/managed-delivery/faq/index.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: single
+title:  "FAQ"
+sidebar:
+  nav: guides
+redirect_from: /reference/managed-delivery/
```

Comment:
This is a new file, right? Do we need this redirect?

Reply:
It's a moved file, but it still doesn't need it.